pyseas/maps/bivariate.py

```diff
@@ -0,0 +1,239 @@
+
```

Comment:
Just taking a quick moment to geek out over how short and succinct you got this code to accomplish something as complex as bivariate mapping. Bravo!

Reply:
😊